setup-deps.sh

```diff
@@ -269,8 +269,7 @@ if command -v jq >/dev/null 2>&1; then
 
   # report but don't stop on error
   set +e
-  npx swagger-cli validate $swagger_file
-  npx @redocly/cli lint $swagger_file
+  REDOCLY_TELEMETRY=off npx @redocly/cli lint $swagger_file
```

Comment:
https://github.com/Redocly/redocly-cli/blob/main/docs/usage-data.md#opt-out-of-data-collection